scripts/variantstore/wdl/GvsCallsetCost.wdl

```diff
@@ -2,11 +2,11 @@ version 1.0
 
 workflow GvsCallsetCost {
     input {
-#        String project_id
-#        String dataset_name
+        String project_name
```

Comment:
Why "project_name"?  All other WDLs use "project_id", I believe.

Reply:
I'll fix this, I actually confused myself with the three different ways of [identifying a Google project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#before_you_begin).